src/main/java/tassist/address/model/person/Github.java

```diff
@@ -31,7 +31,9 @@ public class Github {
     public final String value;
 
     /**
-     * @param github link of the Student
+     * Constructs an {@code Github}.
+     *
+     * @param github A valid Github.
```

Comment:
standardize with the javadoc below?

```suggestion
     * @param github A valid Github link.
```